axelrod/tests/unit/test_shortmem.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for the Cooperator strategy."""
+
+import axelrod
+from .test_player import TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+class TestCooperator(TestPlayer):
+
+    name = "ShortMem"
+    player = axelrod.ShortMem
+    expected_classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Starts by cooperating for the first ten moves.
+        self.responses_test([C], [C] * 4, [D] * 4)
+        self.responses_test([C], [C] * 4 + [D] *6, [D] * 4 + [C] * 6)
+        
+        #Cooperate if in the last ten moves, Cooperations - Defections >= 30
```

Comment:
We're not calculating percentages here. I get that it's pointless to divide by 10 on both sides but it's not an accurate description, and it might be confusing to look at this code in six months. So I'm asking that the comments elaborate more about what the code's intention.